great-idea/css/index.css

```diff
@@ -63,4 +63,149 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* General Styling */
+
+body {
+	width: 80vw;
+	margin: 0 auto;
+}
+
+ul,li,img,div {
+	display: inline-block;
+}
+
+h1 {
+	font-size: 4em;
+	text-align: center;
+}
+
+h2 {
+	font-size: 1.5em;
+}
+
+p {
+	font-size: 18px;
+}
+
+
+/* Header Styling */
+
+nav {
+	display: inline-block;
+	width: 80vw;
+	margin-bottom: 25px;
+	/* position: fixed; 
+	z-index: 1000;
+	background-color: dodgerblue; */
+}
+
+ul {
+	display: inline-block;
+	margin-top: 25px;
+}
+
+ul li {
+	display: inline-block;
+	padding: 0 40px 25px;
+}
```

Comment:
Nice use of vw and em!! For a stretch goal, keep it going here with a %  instead of px to keep everything really responsive! :)